test/e2e/tools/eks-anywhere-test-tool/cmd/root.go

```diff
@@ -31,9 +31,8 @@ func rootPersistentPreRun(cmd *cobra.Command, args []string) {
 }
 
 func initLogger() error {
-	if err := logger.InitZap(logger.ZapOpts{
-		Level: viper.GetInt("verbosity"),
-	}); err != nil {
+
+	if err := logger.InitZap(viper.GetInt("verbosity")); err != nil {
```

Comment:
Doesn't InitZap expect to be passed a ZapOpts?

Reply:
Looks like that signature has changed at least for the verbosity.

Reply:
Yeh, I don't get why that even builds because I'm not seeing it https://github.com/aws/eks-anywhere/blob/main/pkg/logger/zap.go#L26

Reply:
Is there a different logger included there?